### PR TITLE
0.6.0

### PR DESCRIPTION
**Pull request by BenFradet**
- [x] readme doc
- [x] changelog release date
- [x] cla-bot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ The implementation uses a JSON version of the shared 'database' of known referer
 
 The Scala implementation is a core component of [Snowplow][snowplow], the open-source web-scale analytics platform.
 
+### Installation
+
+You can add the following to your SBT config:
+
+```scala
+val refererParser = "com.snowplowanalytics" %% "referer-parser" % "0.6.0"
```
Comment (lukeindykiewicz) [inline README.md]: Artifact is now `scala-referer-parser`

**Review by dilyand — APPROVED** [review]
👍🏻

File: src/test/scala/com/snowplowanalytics/refererparser/CorruptedRefererQuerystringTest.scala [hunk 2]
```diff
@@ -13,31 +13,25 @@
...
-  val expected   = Some(SearchReferer("Google", Some("Psychic Bazaar")))
-
-  val parser = Parser.create[IO](
-    getClass.getResource("/referers.json").getPath
-  ).unsafeRunSync() match {
-    case Right(p) => p
-    case Left(f) => throw f
-  }
+  val resource   = getClass.getResource("/referers.json").getPath
+  val ioParser   = CreateParser[IO].create(resource).unsafeRunSync().fold(throw _, identity)
```
Comment (peel) [inline src/test/scala/com/snowplowanalytics/refererparser/CorruptedRefererQuerystringTest.scala]: as `Either` is a bifunctor (the instance may come from `cats.implicits` though), maybe it's better to just `leftMap` in those cases?

**Review by peel — APPROVED** [review]
Nice! One minor thing that I find worth considering for clarity 👇

File: src/test/scala/com/snowplowanalytics/refererparser/CorruptedRefererQuerystringTest.scala (hunk 2 above)
Comment (dilyand) [reply src/test/scala/com/snowplowanalytics/refererparser/CorruptedRefererQuerystringTest.scala]: I think the advantage of `fold` here is that it always returns a `Parser` at compile time, even if an error might be thrown at run time. That allows us to later do `parser.parse(x)`. 

`leftMap` returns an `Either` and then we have to handle it before we can call `parser.parse(x)`.

**Pull request merged**